crud/src/main/resources/json-schema/crudCommon.json

```diff
@@ -55,17 +55,8 @@
         },
         "execution": {
             "type": "object",
-            "properties": {
-                "timeLimit": {
-                    "type": "integer",
-                    "description": "The upper time limit for the call in milliseconds. If the call does not complete before the limit expires, call fails with partial results written. The return status will contain what items are written."
-                },
-                "asynchronous": {
-                    "type": "integer",
-                    "description": "Given as a time limit in milliseconds. Once that limit is exceeded, the call returns, but execution continues. The return value will contain a task handle that can be used to check for execution status."
-                }
-            },
-            "additionalProperties": false
+            "description": "Execution options captured as key/value pairs.  See documentation on http://lightblue.io for what is supporeted",
+            "additionalProperties": true
```

Comment:
It *is* open ended.

Do you suggest we define the existing properties now, and still allow
additional properties?

On Wed, Mar 9, 2016 at 11:10 AM, Dennis Crissman <notifications@github.com>
wrote:

> In crud/src/main/resources/json-schema/crudCommon.json
> <https://github.com/lightblue-platform/lightblue-core/pull/604#discussion_r55561741>
> :
>
> > @@ -55,17 +55,8 @@
> >          },
> >          "execution": {
> >              "type": "object",
> > -            "properties": {
> > -                "timeLimit": {
> > -                    "type": "integer",
> > -                    "description": "The upper time limit for the call in milliseconds. If the call does not complete before the limit expires, call fails with partial results written. The return status will contain what items are written."
> > -                },
> > -                "asynchronous": {
> > -                    "type": "integer",
> > -                    "description": "Given as a time limit in milliseconds. Once that limit is exceeded, the call returns, but execution continues. The return value will contain a task handle that can be used to check for execution status."
> > -                }
> > -            },
> > -            "additionalProperties": false
> > +            "description": "Execution options captured as key/value pairs.  See documentation on http://lightblue.io for what is supporeted",
> > +            "additionalProperties": true
>
> Why is it preferable to leave this open ended, rather than control what
> properties are possible?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/lightblue-platform/lightblue-core/pull/604/files#r55561741>
> .
>